connection.go

```diff
@@ -1009,3 +1023,16 @@ func (conn *Connection) OverrideSchema(s *Schema) {
 		conn.Schema = s
 	}
 }
+
+// NewStream creates new Stream object for connection.
+//
+// Since v. 2.10.0, Tarantool supports streams and interactive transactions over them.
+// To use interactive transactions, memtx_use_mvcc_engine box option should be set to true.
+// Since 1.7.0
+func (conn *Connection) NewStream() *Stream {
+	atomic.AddUint64(&conn.lastStreamId, 1)
+	return &Stream{
+		Id:   conn.lastStreamId,
+		Conn: conn,
+	}
```

Comment:
A race condition here, a fix:
```suggestion
	next := atomic.AddUint64(&conn.lastStreamId, 1)
	return &Stream{
		Id:   next,
		Conn: conn,
	}
```

Reply:
Done.